packages/neuron-wallet/src/block-sync-renderer/sync/tx-address-finder.ts

```diff
@@ -88,15 +88,20 @@ export default class TxAddressFinder {
     const inputs = this.tx.inputs!.filter(i => i.previousOutput !== null)
     const prefix = NetworksService.getInstance().isMainnet() ? AddressPrefix.Mainnet : AddressPrefix.Testnet
 
+    const outputs = await getConnection()
```

Comment:
If wallet has about 1000 transaction outputs, execute this sql and push to the map will cost about `30 ~ 60ms`, so every 1000 blocks, it will cost about `30 ~ 60s`. It will be much slower than the original version.